modules/virt-booting-efi-mode.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-booting-vms-efi-mode_{context}"]
+= Booting virtual machines in EFI mode
+
+Booting a virtual machine in EFI mode allows for a quicker boot process.
+
+[NOTE]
```

Comment:
If you enable this at the same time as you enable secureBoot, I would note that as a callout in the YAML instead of this note here.